### PR TITLE
LW: task 9, done and dusted

### PR DESCRIPTION
**Pull request by Wainhouse**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -274,3 +272,14 @@ describe.only("POST /api/articles/:article_id/comments", () => {
...
 });
+describe("DELETE api/comments/:comment_id", () => {
+  it('Status 204 "No Content" - Delete Comment', () => {
+    return request(app).delete("/api/comments/5").expect(204);
+  });
+  it('Status 404 "Not Found" - Comment id does not exists', () => {
+    return request(app).delete("/api/comments/1123423").expect(404);
+  });
+  it('Status 400 "Bad Request" - Incorrect comment id', () => {
+    return request(app).delete("/api/comments/ada3q3e").expect(400);
```
Comment (itsaliabdallah) [inline __tests__/app.test.js:283]: nice, for your errors I'd just check you're getting back the message you expect.

File: api/controllers/articles.controller.js [hunk 2]
```diff
@@ -70,3 +73,15 @@ exports.patchArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id: id } = req.params;
+  deleteCommentByID(id)
+    .then(() => {
+      res.status(204).send();
```
Comment (itsaliabdallah) [inline api/controllers/articles.controller.js:81]: nice take a look into res.SendStatus() could save you needing an empty .send()

File: api/models/articles.model.js [hunk 3]
```diff
@@ -69,3 +69,19 @@ exports.addComment = (comment, articleId) => {
       .then((data) => data.rows[0])
 }
 
+exports.deleteCommentByID = (commentId) => {
+  if (!commentId || isNaN(Number(commentId))) {
```
Comment (itsaliabdallah) [inline api/models/articles.model.js:73]: would PSQL throw an error if the comment ID isn't valid?

**Review by itsaliabdallah — APPROVED** [review]
Nice job on this PR!
Check the comment then merge away!

File: api/controllers/articles.controller.js (hunk 2 above)
Comment (Wainhouse) [reply api/controllers/articles.controller.js:81]: Cheers for the tip

File: api/models/articles.model.js (hunk 3 above)
Comment (Wainhouse) [reply api/models/articles.model.js:73]: it would, I'll make the change.

**Pull request merged**